.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+# Contributing
```

Comment:
There's a `Contribute` section in the readme. I'd add a link to this doc in there.

Reply:
I like that idea